### PR TITLE
feat: docstring length

### LINKED ISSUE #44: Parameter for the docstring length
The length of the docstring lines should be a parameter to comply with the PyAnsys documentation.

### PR DESCRIPTION
**Pull request by clatapie**
This PR aims enabling a docstring length parameter.
Closes #44.

### CODE REVIEW

File: src/pyconverter/xml2py/formatter.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"""This module contains the functions to format the generated docstrings with black."""
```
Comment (PipKat) [inline src/pyconverter/xml2py/formatter.py:1]: Link to the repo for this codes style tool to provide clarity?

**Review by RobPasMue — APPROVED** [review]

File: src/pyconverter/xml2py/cli.py [hunk 2]
```diff
@@ -50,6 +56,14 @@ def create_package(xml_path=None, functions_path=None, target_path=None, templat
         The default value is the ``_package`` directory accessible in the
         `PyConverter-XML2Py GitHub repository <https://github.com/ansys/pyconverter-xml2py/tree/main/_package>`_.
 
+    run_black: bool, optional
+        Whether to run black CLI on the autogenerated package source code.
```
Comment (PipKat) [inline src/pyconverter/xml2py/cli.py:60]:
```suggestion
        Whether to run Black CLI on the autogenerated package source code.
```
I checked their doc, and they use an initial capital letter in references to the product name.

File: src/pyconverter/xml2py/cli.py [hunk 3]
```diff
@@ -50,6 +56,14 @@ def create_package(xml_path=None, functions_path=None, target_path=None, templat
...
+        The default value is False.
```
Comment (PipKat) [inline src/pyconverter/xml2py/cli.py:61]:
```suggestion
        The default value is ``False``.
```

File: src/pyconverter/xml2py/cli.py [hunk 4]
```diff
@@ -50,6 +56,14 @@ def create_package(xml_path=None, functions_path=None, target_path=None, templat
...
+
+    max_docstring_length: int, optional
+        Maximum length of the generated docstrings.
+        The default value is 100.
```
Comment (PipKat) [inline src/pyconverter/xml2py/cli.py:65]:
```suggestion
        The default value is ``100``.
```

File: src/pyconverter/xml2py/cli.py [hunk 5]
```diff
@@ -126,6 +142,20 @@ def version():
...
+    type=click.INT,
+    default=100,
+    help="Maximum length of the generated docstrings.",
+)
+@click.option(
+    "-b",
+    "--run-black",
+    type=click.BOOL,
+    default=False,
+    help="Whether to run black CLI on the autogenerated package source code. The default value is False.",  # noqa : E501
```
Comment (PipKat) [inline src/pyconverter/xml2py/cli.py:157]:
```suggestion
    help="Whether to run Black CLI on the autogenerated package source code. The default is 'False'.",  # noqa : E501
```

**Pull request merged**